test/Poseidon/EntitiesListSpec.hs

```diff
@@ -233,160 +238,130 @@ testResolveEntityIndices =
             , Exclude (Pac (PacNameAndVersion "Pac1" (Just $ makeVersion [1,0,0])))
-            ] indInfo `shouldBe` ([], [2])
+            ] indInfo `shouldBe` [2]
         resolveEntityIndices [
-              Include (Ind (SpecificInd $ IndividualInfo "Ind1" ["Pop1"] (PacNameAndVersion "Pac1" (Just $ makeVersion [2,0,0]))))
-            , Include (Ind (SpecificInd $ IndividualInfo "Ind1" ["Pop1"] (PacNameAndVersion "Pac1" (Just $ makeVersion [1,0,0]))))
+              Include (SpecificInd "Ind1" "Pop1" (PacNameAndVersion "Pac1" (Just $ makeVersion [2,0,0])))
+            , Include (SpecificInd "Ind1" "Pop1" (PacNameAndVersion "Pac1" (Just $ makeVersion [1,0,0])))
             , Exclude (Pac (PacNameAndVersion "Pac1" Nothing))
-            , Include (Ind (SimpleInd "Ind4"))
-            ] indInfo `shouldBe` ([], [5])
+            , Include (Ind "Ind4")
+            ] indInfo `shouldBe` [5]
 
-testJSON :: Spec
-testJSON =
-    describe "Poseidon.EntitiesList.ToJSON" $ do
-        it "should encode entities correctly to JSON" $ do
-            encode (Ind (SimpleInd "Ind1")) `shouldBe`
-                    "\"<Ind1>\""
-            encode (Ind (SpecificInd $ IndividualInfo "a" ["b"] (PacNameAndVersion "c" Nothing))) `shouldBe`
-                    "\"<c:b:a>\""
-            encode (Group "Group1") `shouldBe`
-                    "\"Group1\""
-            encode (Pac (PacNameAndVersion "Pac1" Nothing)) `shouldBe`
-                    "\"*Pac1*\""
-            encode (Exclude (Ind (SimpleInd "Ind1"))) `shouldBe`
-                    "\"-<Ind1>\""
-            encode (Exclude (Ind (SpecificInd $ IndividualInfo "a" ["b"] (PacNameAndVersion "c" Nothing)))) `shouldBe`
-                    "\"-<c:b:a>\""
-            encode (Exclude (Group "Group1")) `shouldBe`
-                    "\"-Group1\""
-            encode (Exclude (Pac (PacNameAndVersion "Pac1" Nothing))) `shouldBe`
-                    "\"-*Pac1*\""
-        it "should decode entities correctly from JSON" $ do
-            decode "\"<Ind1>\""   `shouldBe`
-                    Just (Ind (SimpleInd "Ind1"))
-            decode "\"<c:b:a>\""  `shouldBe`
-                    Just (Ind (SpecificInd $ IndividualInfo "a" ["b"] (PacNameAndVersion "c" Nothing)))
-            decode "\"Group1\""   `shouldBe`
-                    Just (Group "Group1")
-            decode "\"*Pac1*\""   `shouldBe`
-                    Just (Pac (PacNameAndVersion "Pac1" Nothing))
-            decode "\"-<Ind1>\""  `shouldBe`
-                    Just (Exclude (Ind (SimpleInd "Ind1")))
-            decode "\"-<c:b:a>\"" `shouldBe`
-                    Just (Exclude (Ind (SpecificInd $ IndividualInfo "a" ["b"]  (PacNameAndVersion "c" Nothing))))
-            decode "\"-Group1\""  `shouldBe`
-                    Just (Exclude (Group "Group1"))
-            decode "\"-*Pac1*\""  `shouldBe`
-                    Just (Exclude (Pac (PacNameAndVersion "Pac1" Nothing)))
+-- testJSON :: Spec
```

Comment:
Why did we have the ToJSON instance of `SignedEntity`? I guess you realized that we don't use it any more (?), removed it and then deactivated these tests here, right?

Maybe we should rewrite these tests to cover the show instance now? That would be a good use for this code. The `decode` tests can be removed, as we already have tests for `readEntitiesFromString` above.

Reply:
Can't exactly remember, but indeed I think I removed JSON instances where I didn't see use for. But you're right, we could test the Show instances.

Reply:
Done.